src/mlpack/methods/ann/layer/transposed_convolution_impl.hpp

```diff
@@ -280,9 +340,16 @@ void TransposedConvolution<
   ar & BOOST_SERIALIZATION_NVP(inputHeight);
   ar & BOOST_SERIALIZATION_NVP(outputWidth);
   ar & BOOST_SERIALIZATION_NVP(outputHeight);
+  ar & BOOST_SERIALIZATION_NVP(aW);
+  ar & BOOST_SERIALIZATION_NVP(aH);
 
   if (Archive::is_loading::value)
+  {
     weights.set_size((outSize * inSize * kW * kH) + outSize, 1);
+
+    aW = (outputWidth + kW - 2 * padW - 2) % dW;
```

Comment:
Ah, actually, this is a good point.  If we are serializing `aW` and `aH`, then we have to handle reverse compatibility.  (That's not hard but it's just a little bit tedious.)  Since as far as I can tell `aW` and `aH` are always computed, I'd suggest that we just remove the two lines `ar & BOOST_SERIALIZATION_NVP(aw);` and `ar & BOOST_SERIALIZATION_NVP(aH);` above.